.circleci/config.yml

```diff
@@ -364,7 +507,9 @@ workflows:
       # Build the python package
       - package-back:
           requires:
+            - test-front
             - test-back
+            - build-front-production
```

Comment:
Did you check that your package contains front-end production builds?

Reply:
Yes I just did :+1: 
In the artifact of the following circleCI jobs : https://circleci.com/gh/openfun/ashley/1509#artifacts/containers/0